docs/Reference/Glossary.md

```diff
@@ -20,19 +20,18 @@
 * **version**: A version, which corresponds to a vertical slice of tasks on the waterfall, is all tasks for a given commit or patch build.
 * **working directory**: The working directory is the temporary directory which Evergreen creates to run a task. It is available as [an expansion](../Project-Configuration/Project-Configuration-Files#default-expansions).
 
-
 ## Requesters
 
-The requester is field in the version/patch document that indicates how it was created. These are secure, immutable fields that are set upon creation.
+The requester is a field in the version/patch document that indicates how it was created. These immutable values are set upon creation. They are used to provide elevated permissions to certain versions/patches. For example, a project can have [admin only](../Project-Configuration/Project-and-Distro-Settings.md#variables) variables that only get provided to versions/patches by admins or from mainline commits. They can also be used with [ec2.assume_role](../Project-Configuration/Project-Commands.md#ec2assume_role) to allow specific requesters to assume a role (e.g.mainline commits) and prevent others from not (e.g. CLI requests.).
```

Comment:
On top of permissions differences, they're also used for other purposes like to run different sets of tasks depending on the source of the version.

Reply:
Added!